app/Http/Controllers/UsersController.php

```diff
@@ -334,7 +331,7 @@ private function recentKudosu($user, $perPage = 5, $offset = 0)
         );
     }
 
-    private function mostPlayedBeatmapsets($user, $perPage = 5, $offset = 0)
+    private function mostPlayedBeatmapsets($user, $perPage = 6, $offset = 0)
```

Comment:
not sure if it would make sense for those to be `original_count + 1`, for some more code readability